{"name": "test_login_0_failed_wrong_datas[data0]", "status": "passed", "attachments": [{"name": "log", "source": "dfa277c5-c50e-482c-a808-9f42dadd784c-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'user': '18684720553', 'passwd': '', 'check': '请输入密码'}"}], "start": 1608299392841, "stop": 1608299393141, "uuid": "d4e36f96-4754-451d-91de-ba29812c91cb", "historyId": "98ee763b71a2e5d2e8433e177b71f895", "testCaseId": "735933ecd585cb26d239d15346a80fa1", "fullName": "testcase.test_1login_pytest.TestLogin#test_login_0_failed_wrong_datas", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page')"}, {"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_1login_pytest"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-J6CF24D"}, {"name": "thread", "value": "59000-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_1login_pytest"}]}